examples/text_summarization/extractive_summarization_cnndm_distributed_train.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Did you eventually figure out why one of you nodes is always ahead of the other one?

Reply:
The gradient sync is asynchronous

Reply:
Is there a reference documentation?

Reply:
https://pytorch.org/tutorials/intermediate/ddp_tutorial.html#skewed-processing-speeds